apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -33,6 +33,29 @@ import { UploadPoliceCaseFileResponse } from './models/uploadPoliceCaseFile.resp
 import { policeModuleConfig } from './police.config'
 import { PoliceCaseInfo } from './models/policeCaseInfo.model'
 
+function getChapter(category?: string): number | undefined {
+  if (!category || category.length < 2) {
+    return undefined
+  }
+
+  switch (category.substring(0, 2)) {
```

Comment:
This is a bit too hard coded in my opinion. Consider casting the chapter number to an int and return `chapterNumber - 1`. If the type casting fails or returns a `NaN`, return `undefined`. That way we don't need to change this code if a chapter is added or removed in the future.